src/lib/components/Popover/README.mdx

```diff
@@ -0,0 +1,294 @@
+  return (
+    <div
+      style={{
+        display: 'grid',
+        placeContent: 'center',
+        minWidth: '20rem',
+        minHeight: '10rem',
+      }}
+    >
+      <div style={{position: 'relative'}}>
+        <Button
+          aria-describedby={isPopoverOpen ? 'my-popover' : undefined}
+          label="Want to see a popover? Click me!"
+          onClick={() => setIsPopoverOpen(!isPopoverOpen)}
+        />
+        {isPopoverOpen && (
+          <Popover id="my-popover">
+            Hello there!
+          </Popover>
+        )}
+      </div>
+    </div>
+  );
+}}
+</Playground>
+
+See [API](#api) for all available options.
+
+## Placement
+
+By default, Popover is placed relative to the closest parent element with
+`position: relative` or `position: absolute`. Available placements are: top,
+right, bottom, and left. Additionally, all basic placements can be aligned to
+the center (default, no suffix), start (e.g. `top-start`), or end (e.g.
+`bottom-end`). Check Popover [API](#api) for the complete list of accepted
+values.
+
+<Playground>
+{() => {
+  const [align, setAlign] = React.useState('');
+  // All inline styles in this example are for demonstration purposes only.
+  return (
+    <>
+      <Toolbar align="baseline">
+        <ToolbarItem>
+          <span id="alignment-options-label">Alignment:</span>
+        </ToolbarItem>
+        <ToolbarItem>
+          <ButtonGroup aria-labelledby="alignment-options-label">
+            <Button
+              color={align === '-start' ? 'dark' : 'primary'}
+              label="start"
+              onClick={() => setAlign('-start')}
+            />
+            <Button
+              color={align === '' ? 'dark' : 'primary'}
+              label="center"
+              onClick={() => setAlign('')}
+            />
+            <Button
+              color={align === '-end' ? 'dark' : 'primary'}
+              label="end"
+              onClick={() => setAlign('-end')}
+            />
+          </ButtonGroup>
+        </ToolbarItem>
+      </Toolbar>
+      <div
+        style={{
+          display: 'grid',
+          placeContent: 'center',
+          minWidth: '20rem',
+          minHeight: '15rem',
+        }}
+      >
+        <div style={{ position: 'relative' }}>
+          <Placeholder bordered aria-describedby="my-popover-top">
+            Popovers
+            <br />
+            all day long…
+          </Placeholder>
+          <Popover id="my-popover-top" placement={`top${align}`}>
+            Top side
+          </Popover>
+          <Popover id="my-popover-right" placement={`right${align}`}>
+            Right side
+          </Popover>
+          <Popover id="my-popover-bottom" placement={`bottom${align}`}>
+            Bottom side
+          </Popover>
+          <Popover id="my-popover-left" placement={`left${align}`}>
+            Left side
+          </Popover>
+        </div>
+      </div>
+    </>
+  );
+}}
+</Playground>
+
+## Advanced Positioning
```

Comment:
I'm confused here. I thought that when I use the `floating-ui` library I don't have to worry about placement at all. For example, I was expecting that as I turn tablet or scroll the library would do its best to keep the popover visible.

In the demo It seems to me, that the Basic and Advanced examples behave the same.

Reply:
Thanks for pointing it out. I tried to showcase such an example but it imposes so many questions and gotchas (beware of, do this, don't do that…) that I ended up having just the simplest example possible.

I will give it one more try.

Reply:
Personally, I am OK with present state. We just need to show how it is possible to implement it with some library (in our case - recommended Floating UI), but I do not see any benefit from showing advanced example. Different people needs deferent behavior and it is up to them how they want to have their instance configured.

Reply:
OK, no more tries.

Reply:
Sorry, but I'm still confused.

Does using floating-ui actually change the behaviour? If yes, the example should reflect that and make it obvious. If not why bother mentioning it?

Reply:
Also I think we should change the section title from `Advanced Positioning` to `Positioning with External Library`

Reply:
Finally I succeeded making a better example that is not overwhelming yet demonstrates the idea. What do you think now?

<img width="707" alt="Snímek obrazovky 2022-06-25 v 16 22 16" src="https://user-images.githubusercontent.com/5614085/175777956-ed30744a-a29a-4db0-9881-43ff70c00422.png">

Reply:
It's much better now! I think that @mbohal will be OK with that :)

Reply:
Yes, this is waaaaaay better.